task_processing/plugins/smartstack.py

```diff
@@ -0,0 +1,34 @@
+import yaml
+
+
+YOCALHOST_IP = '169.254.255.254'
+TASKPROC_SERVICE_FILE = '/nail/etc/services/task_processing/smartstack.yaml'
```

Comment:
I agree for the services file, but the ip is link-local and useless to outside Yelp. I think it's okay?